src/snowflake/connector/cursor.py

```diff
@@ -464,9 +468,7 @@ def _execute_helper(
             self._is_file_transfer = _is_put_get
         else:
             # or detect it.
-            self._is_file_transfer = self.PUT_SQL_RE.match(
-                query
-            ) or self.GET_SQL_RE.match(query)
+            self._is_file_transfer = bool(self.get_file_transfer_type(query))
```

Comment:
The server side response is different for a PUT/GET command, but I don't see much value in modifying these regexes to detect PUT/GET commands in multi-statement since they are not supported for multi-statements anyways. I think we should make very clear in the docs that PUT/GET commands are not supported for multi-statements, and leave the existing code as is. This would exactly match the implementation in JDBC.